### ISSUE TITLE
Problem sending file

### ISSUE THREAD

**Issue by luishens01** [taverntesting/tavern#93]
Hi, 
we are currently learning Tavern for Rest testing and I have a problem creating a test.

This is an example of the working curl that I need to create in Tavern:

`curl -f -v -i -X POST -H "Authorization:Bearer XXX...XXXX" -F package=@/dir1/dir2/MY_FILE http://url`

and this is the way we are trying to recreate it:

```yaml
  _- name: upload_package
    request:
      url: http://url
      method: POST
      headers:
        authorization: "Bearer {test_login_token:s}"
        Content-Type: "multipart/form-data;
      files: 
        package: "/dir1/dir2/MY_FILE"
    response:
      status_code: 201_
```

This is not working for us and have been trying a lot of combinations with the content-types in headers.

Could you guide us in how we can achieve this ?

Thanks in advance
Luis

**Comment by michaelboulton** [comment]
When you say it's not working, do you mean that it's not sending the file with the correct content type or Tavern is giving an error of some sort? Could give you the output so we can look at it?

**Comment by Daishi1223** [comment]
When we set header `Authorization`, received header `Content-Type` will be `application/json`.

**Comment by tjacoby** [comment]
I have the same problem. For file uploads the `Content-Type` header has to include the boundary information (see https://tools.ietf.org/html/rfc1867#section-6). As @Daishi1223 mentioned the Content-Type header is overridden with `application/json` whenever additional headers are passed.

**Comment by ChristianSauer** [comment]
I have the same problem and it's a showstopper

**Comment by michaelboulton** [comment]
This should be fixed in 0.9.8 whenever Travis deploys it

**Comment by michaelboulton** [comment]
Closing this as it should now be fixed.
